Gruntfile.js

```diff
@@ -60,17 +60,6 @@ module.exports = function(grunt) {
     grunt.loadNpmTasks('grunt-contrib-uglify');
     grunt.loadNpmTasks('grunt-contrib-watch');
 
-    // Custom task
-    grunt.registerTask('sync_versions', 'Keeps versions in sync between NPM and Bower', function(){
-        var bower = {
-            name: pkg.name,
-            author: pkg.author,
-            version: pkg.version,
-            main: 'dist/ripple.min.js'
-        };
-        fs.writeFileSync('bower.json', JSON.stringify(bower, null, "\t"));
-    });
-
-    grunt.registerTask('default', ['jshint', 'uglify', 'cssmin', 'sync_versions']);
-    grunt.registerTask('develop', ['jshint', 'uglify', 'cssmin', 'sync_versions', 'watch']);
```

Comment:
Bower use auto version by github tags. So this is unnecessary. And bower must contain more info. And I add this info.

I understand grunt just know))